webapp/graphite/metrics/views.py

```diff
@@ -290,6 +295,16 @@ def tree_json(nodes, base_path, wildcards=False):
   return results
 
 
+def nodes_by_position(matches, position):
+  found = set()
+
+  for metric in matches:
+    nodes = metric.path.split('.')
+    found.add(nodes[position])
+    results = { 'nodes' : sorted(found) }
```

Comment:
@obfuscurity dedent this line one level? It's only needed at after the loop is done. After that, LGTM :+1: :)